### ISSUE TITLE
*:before, *:after causes errors

### ISSUE THREAD

**Issue by dirkluijk** [dompdf/dompdf#758]
Hi,

I want to generate a PDF with the same stylesheet I use on a website. This stylesheet also contains a CSS reset with the following snippet:

```
*,
*:before,
*:after {
  -webkit-box-sizing: border-box;
     -moz-box-sizing: border-box;
          box-sizing: border-box;
}
```

This is causing the following error: 
`Call to a member function prepend_child() on a non-object in \path\to\dompdf\dompdf\include\frame_tree.cls.php on line 232`. It is trying to apply a style to the `<head>` DomNode, which is not available in the loaded tree.

It looks like it's not checked whether the element is available when applying `:before` and `:after` selectors.

`* { some: css; }` works fine.
`p:before { some: css; }` works fine.
`head:before { some: css; }` -> ERROR
`*:before { some: css; }` -> ERROR

Thanks.

**Comment by cigzigwon** [comment]
Boostrap 3 works okay for me.... the prepend child call is tacking in extra cells to my table but I'm not sure why. When I comment out the prend function altogether in the frame class my issue goes away.

I'm not extending the code so the workaround is this:
<pre>
<code>
<tr><td style="border:none;background:#fff"></td></tr>
</code>
</pre>

Which I'm prepending my own child to bork the invisible first row... I also had to toss out the thead and tbody tags because the exponentiate the issue.

Hope this helps for you BS3 people! I'm not sure if BS2 even works...

**Comment by pixelchutes** [comment]
I've been able to reproduce this issue by with something as simple as:

```html
<!DOCTYPE html>
<html>
    <head>
        <meta charset="UTF-8">
        <title>Psuedo Issue</title>
        <style>
            :before{}
        </style>
    </head>
    <body>
        <input type="text" value="This text gets hidden">
    </body>
</html>
```

### WITHOUT PSEUDO `:before` or `:after`:

![psuedo_issue_without](https://cloud.githubusercontent.com/assets/352182/22954188/45739522-f2d2-11e6-9cbf-386af3d3162f.jpg)

### WITH PSEUDO `:before` or `:after`:

![psuedo_issue_with](https://cloud.githubusercontent.com/assets/352182/22954160/1e1f5f24-f2d2-11e6-97bc-c9103e6274da.jpg)

**Comment by bsweeney** [comment]
@dirkluijk your particular sample appears to be sufficiently addressed in the upcoming (0.8.0) release.
